### ISSUE TITLE
Navigational improvements in the browse section

### ISSUE THREAD

**Issue by peetucket** [sul-dlss/rialto-webapp#173]
When viewing a list of publications in browse, if click on a publication to view details, you need to use the browser button to get back to search results.  Some navigational breadcrumbs would help figure out where you are.

**Comment by jcoyne** [comment]
I don't understand how breadcrumbs would work if you follow links from "search" -> "publication 1" -> "author" -> "publication 2" -> "grant" -> "organization".  What should the breadcrumbs look like on each page?

**Comment by peetucket** [comment]
Agreed, breadcrumbs is probably the wrong navigational pattern, this is something that @astridu could think about (though this is not a high priority for this workcycle either).

**Comment by astridu** [comment]
Would the user be interested in accessing the intermediary pages (between when she first searched and the current page she's on), or is she just interested in getting back to her original search?

**Comment by mjgiarlo** [comment]
Keep in backlog or move to ready, @peetucket?

**Comment by peetucket** [comment]
let's backlog this for now until we have more feedback on how the browse is likely to be used
